CHANGELOG.md

```diff
@@ -42,6 +42,60 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 - Added field for changing execution engine used in a build via the
   "RUN ALL OPTIONS" modal. (#122)
 
+- Removed unused configs, specified in `assets/config.json`, and
+  `"Environment"` object in `assets/config.json` and inlined their configs
+  instead, as well as renamed `Name` to `name` and `IsProduction` to
+  `production`: (#127)
+
+  ```diff
+  --- a/assets/config.json
+  +++ b/assets/config.json
+   {
+  -  "Environment": {
+  -    "Name": "lorem-ipsum",
+  -    "IsProduction": true
+  -  },
+  +  "name": "lorem-ipsum",
+  +  "production": true,
+
+  -  "UpdateLatency": 20000,
+  -  "UpdateFrequency": 30000,
+
+     "BackendUrls": {
+       "Api": "/api",
+       "Gitlab": "/import",
+       "Github": "/import",
+       "AzureDevops": "/import",
```

Comment:
I want to say there is a potential issue here with how this works with the proxy setup for local dev when using the different varieties of angular proxy configs. Probably nothing that needs consideration here, but consider it a <danger> heads up for later. it should be relativly simple to debug once the error occurs by looking at dev tools in the browser.